WorkshopAutoTranslation/WorkShopTranslationV2/Program.cs

```diff
@@ -0,0 +1,123 @@
+﻿using Azure;
+using Azure.AI.Inference;
+
+namespace WorkShopTranslationV2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            #region Secrets
+            var endpoint = new Uri("https://models.inference.ai.azure.com");
+            var credential = new AzureKeyCredential("Enter your Pat token");
```

Comment:
I wonder if we can work with Jeremiah to get an Azure Key Vault instance set up and we can store it there, instead of hard-coding the Pat token?